### PR TITLE
update allowed sub types

### PR DESCRIPTION
**Pull request by urosran**

### CODE REVIEW

**Comment by urosran** [comment]
#sign-off

**Comment by urosran** [comment]
#signoff

**Comment by Court72** [comment]
@vladai78

Can you review the proposed changes? Note that the validation warnings must be resolved before merging.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/managed-instance/resource-limits.md [hunk 1]
```diff
@@ -168,6 +168,9 @@ SQL Managed Instance currently supports deployment only on the following types o
 - [Enterprise Dev/Test](https://azure.microsoft.com/offers/ms-azr-0148p/)
 - [Pay-As-You-Go Dev/Test](https://azure.microsoft.com/offers/ms-azr-0023p/)
 - [Subscriptions with monthly Azure credit for Visual Studio subscribers](https://azure.microsoft.com/pricing/member-offers/credit-for-visual-studio-subscribers/)
+- [Free Trial](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0044p/)
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/resource-limits.md]:
```suggestion
- [Free Trial](https://azure.microsoft.com/pricing/offers/ms-azr-0044p/)
```

File: azure-sql/managed-instance/resource-limits.md [hunk 2]
```diff
@@ -168,6 +168,9 @@ SQL Managed Instance currently supports deployment only on the following types o
...
+- [Azure For Students](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0170p/)
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/resource-limits.md]:
```suggestion
- [Azure For Students](https://azure.microsoft.com/pricing/offers/ms-azr-0170p/)
```

File: azure-sql/managed-instance/resource-limits.md [hunk 3]
```diff
@@ -168,6 +168,9 @@ SQL Managed Instance currently supports deployment only on the following types o
...
+- [Azure In Open](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0111p/)
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/resource-limits.md]:
```suggestion
- [Azure In Open](https://azure.microsoft.com/pricing/offers/ms-azr-0111p/)
```

**Comment by vladai78** [comment]
#sign-off

**Pull request merged**